vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -136,75 +131,3 @@ def plot(
         # TODO Tentative for integration test
         if self._return_all_text:
             return output_dict
-
-
-def _debug_helper(
-    code_string: str, max_debug_retry: int, fix_chain: Callable, df: pd.DataFrame = None
-) -> Dict[bool, str]:
-    """Debugging helper."""
-    # TODO plug logic back into component
-    retry_success = False
-    last_exception = None
-    for attempt in range(max_debug_retry):
-        try:
-            _exec_code(code=code_string, local_args={"df": df}, is_notebook_env=_is_jupyter())
-            retry_success = True
```

Comment:
Do we want to move the `_is_jupyter()` check outside the loop? I assume it will not change and only needs to be called once? Otherwise leave in